src/capwap_loc_provider_http.erl

```diff
@@ -0,0 +1,65 @@
+%% Copyright (C) 2013-2023, Travelping GmbH <info@travelping.com>
+%%
+%% This program is free software: you can redistribute it and/or modify
+%% it under the terms of the GNU Lesser General Public License as
+%% published by the Free Software Foundation, either version 3 of the
+%% License, or (at your option) any later version.
+%%
+%% This program is distributed in the hope that it will be useful,
+%% but WITHOUT ANY WARRANTY; without even the implied warranty of
+%% MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+%% GNU Lesser General Public License for more details.
+%%
+%% You should have received a copy of the GNU Lesser General Public License
+%% along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+-module(capwap_loc_provider_http).
+
+-compile({parse_transform, cut}).
```

Comment:
What for this needed here?

Reply:
It may be a leftover from the previous approach based on aaa handlers.